clients/src/main/resources/common/message/FetchSnapshotResponse.json

```diff
@@ -51,8 +51,8 @@
           "about": "The total size of the snapshot." },
         { "name": "Position", "type": "int64", "versions": "0+",
           "about": "The starting byte position within the snapshot included in the Bytes field." },
-        { "name": "Bytes", "type": "bytes", "versions": "0+", "zeroCopy": true,
-          "about": "Snapshot data." }
+        { "name": "Bytes", "type": "records", "versions": "0+",
```

Comment:
I had a previous comment about changing the name here from "Bytes" to "UnalignedRecords." What do you think about it?

Reply:
I have a question here, should we bump the version when changing the name?

Reply:
We do not need to. The name is not part of the serialization. It only affects the generated code.